packages/manager/apps/catalog-revamp/tsconfig.json

```diff
@@ -20,6 +20,7 @@
     "baseUrl": ".",
     "paths": {
       "@/*": ["./src/*"],
+      "@mocks/*": ["./mocks/*"],
```

Comment:
we don't need that, just use a relative path

Reply:
Removed.